lib/JointspaceClient.js

```diff
@@ -469,13 +469,18 @@ class JointspaceClient {
         return this.request('audio/volume', 'GET');
     }
 
-    setVolume(volume) {
+    setVolume(volume, mute) {
+        if (typeof mute === 'undefined') {
+            mute = false;
+        }
         return this.request('audio/volume', 'POST', {
             current: volume,
-            muted: false
+            muted: mute
```

Comment:
I think this would make the `if (typeof mute === 'undefined')` statement obsolete.

```suggestion
            muted: mute || false
```